### ISSUE TITLE
test migrating data to PUL postgres instances

### ISSUE THREAD

**Issue by thatbudakguy** [Princeton-CDH/cdh-web#227]
PUL postgres staging runs v12, but prod runs v10, so we need to ensure that the data migration is compatible with postgres 10.

Francis recommended [the PGLoader tool](https://pgloader.readthedocs.io/en/latest/ref/mysql.html) to do this.

**Comment by thatbudakguy** [comment]
tested locally using mysql and postgres containers; the pgloader tool seems to work great without any configuration. i installed the tool using `brew`. the syntax I used to copy between the two databases was:
```sh
$ pgloader mysql://user:pass@hostname:3306/dbname postgresql://user:pass@hostname:5432/dbname
```
the run completed in 13s and reported 0 errors. I did see a lot (~25?) of warnings noting that the names of certain autogenerated identifiers (i think field names?) would be truncated due to length:
```sh
2020-11-09T20:47:49.452901Z WARNING PostgreSQL warning: identifier "idx_16709_projects_projectresou_project_id_f9500731_fk_projects_project_id" will be truncated to "idx_16709_projects_projectresou_project_id_f9500731_fk_projects"
```
...but this didn't seem to cause any problems. reading some docs, it appears that Django intelligently detects the truncation and knows to use the shorter form when the postgres adapter is used for the ORM.
